lib/controllers/usuario_controller.test.js

```diff
@@ -111,4 +111,19 @@ describe('Usuario controller', () => {
       });
     });
   });
+
+  describe('Actualización de un usuario', () => {
+    it('Al actualizar un valor no cambia la contraseña', async () => {
+      const nuevaFecha = new Date();
+      const usuarioAntesDeActualizar = await Usuario.findByPk(informatico.id);
+
+      await informatico.update({ fechaSincronizacionGuarani: nuevaFecha });
+
+      expect(informatico.dataValues).toMatchObject({
+        ...usuarioAntesDeActualizar.dataValues,
+        fechaSincronizacionGuarani: nuevaFecha,
+        updatedAt: informatico.updatedAt,
+      });
```

Comment:
Mmm, este test prueba mucho más que lo que debería. Yo probaría únicamente lo que el nombre del test dice:

```suggestion
      expect(informatico.contrasenia).toEqual(usuarioAntesDeActualizar.contrasenia);
```

Reply:
Además, este test no tiene que ver con el controller sino con el modelo puro. Debería ir a un archivo `usuario.test.js` dentro de la carpeta de los modelos.